tdesign/src/Layout/PaddedSingleCol/PaddedSingleCol.tsx

```diff
@@ -31,10 +29,10 @@ const PaddedSingleCol = ({
   const outerContainerStyle: SxProps<Theme> = {
     ...theme.grids.threeCol,
     ".left-spacer": {
-      minWidth: leftPadding ?? theme.constants.leftMarginInsideLogo,
+      width: "max(25px, calc((100vw - 784px) / 2))",
```

Comment:
Can you explain the 784px magic number here?

I find that creating a const variable and interpolating it here is usually sufficient for explanation, like:

```ts
const theVariableNameThatExplainsWhatThisIs = "784px";
```

and then use it like

```suggestion
      width: `max(25px, calc((100vw - ${theVariableNameThatExplainsWhatThisIs}) / 2))`,
```

I didn't check, but if that value corresponds to some theme constant (like a breakpoint value), let's use it directly - it's easier to see the connection by using a variable rather than the number itself

Reply:
Unfortunately not a theme constant :( 784px was the width in the Figma and I took that to be a max width to prevent the content from stretching out too wide. Def something I want to bring up with Flambee